app/assets/javascripts/file-saver/CHANGELOG.md

```diff
@@ -0,0 +1,60 @@
+# Changelog
```

Comment:
nm

Reply:
Do you have redis/sidekiq running locally?